test/test_images/performance/main.go

```diff
@@ -31,6 +32,8 @@ import (
 )
 
 // flags for the image
+type Tags []string
```

Comment:
Do you expect this type to implement more methods besides `String` and `Set`? As written, I don't think the type is necessary and using `[]string` everywhere would be better readability.